src/config_cache.cpp

```diff
@@ -78,7 +78,6 @@ void config_cache::clear_defines()
 #ifdef __APPLE__
 	defines_map_["APPLE"] = preproc_define();
 
-#define MOUSE_TOUCH_EMULATION
```

Comment:
I assume this will ultimately be a compiled-time flag? Like `-DMOUSE_TOUCH_EMULATION` or whatever.